static/brothers/index.html

```diff
@@ -8,16 +8,16 @@
     <title>Brothers &ndash; Alpha Epsilon of Phi Kappa Sigma</title>
     
     <!-- Import Pure components -->
-    <link rel="stylesheet" href="https://yui.yahooapis.com/pure/0.6.0/pure-min.css">
+    <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css" integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous">
     
     <!--[if lte IE 8]>
 
-        <link rel="stylesheet" href="https://yui.yahooapis.com/pure/0.6.0/grids-responsive-old-ie-min.css">
+        <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/grids-responsive-old-ie-min.css" integrity="sha384-sGrAmFbC85ibuG+7mNPSCxr6U5S3ObASQDpRTlck+CBI4SRyHG9lIto9YyA92YG9" crossorigin="anonymous">
```

Comment:
I mean, I'm open to continue using yahoo hosting, what's wrong with that? 😝

Reply:
![image](https://user-images.githubusercontent.com/1095869/34228246-1a29ce08-e59e-11e7-86e0-9ebc2d212f59.png)

...yeah 😝